lib/Target/Python/TranslateToPython.cpp

```diff
@@ -73,7 +73,7 @@ inline LogicalResult interleaveCommaWithError(const Container container,
 namespace {
 /// Emitter that uses dialect specific emitters to emit Python code.
 struct PythonEmitter {
-  explicit PythonEmitter(raw_ostream &os) : os(os) {
+  explicit PythonEmitter(raw_ostream &os) : os(os), inExpressionContext(false) {
```

Comment:
:warning: **cppcoreguidelines\-use\-default\-member\-init** :warning:
use default member initializer for `` inExpressionContext ``
```suggestion
  explicit PythonEmitter(raw_ostream &os) : os(os), {
```